cmd/loadtest/init.go

```diff
@@ -44,7 +45,7 @@ func createChannels(admin *userentity.UserEntity, numChannels int) error {
 	channelTypes := []string{"O", "P"}
 
 	for i := 0; i < numChannels; i++ {
-		team, err := admin.Store().RandomTeam()
+		team, err := admin.Store().RandomTeam(store.SelectMemberOf | store.SelectNotMemberOf)
```

Comment:
I think @agnivade meant actually defining a `SelectAny` to be `SelectMemberOf | SelectNotMemberOf`.

Reply:
We can define `SelectAny` effectively it will be the same.